src/main/app.py

```diff
@@ -0,0 +1,43 @@
+import logging
+from flask import Flask, request
+
+app = Flask(__name__)
+app.logger.setLevel(logging.INFO)
+NAME = "Bharadwaj"
+
+name = None
+
+
+@app.route('/', methods=['GET'])
+async def index():
```

Comment:
if you don't know how to use async, delete it from here and remove the `[async]` from `requirements.txt`